test/parallel/test-child-process-spawnsync-maxbuf.js

```diff
@@ -16,7 +16,7 @@ const args = [
 {
   const ret = spawnSync(process.execPath, args, { maxBuffer: 1 });
 
-  assert.ok(ret.error, 'maxBuffer should error');
+  assert.ok(ret.error, `instead of an error there is ${ret.error}` );
```

Comment:
Since `ret` is returned by `spawnSync`, `ret.error` can only be either an `Error` or `undefined`, so it it fails this would always be "instead of an error there is undefined", which does not seem very helpful... I think changing the message to `child process does not error when maxBuffer is set to 1` would be easier to understand.

Reply:
Good point, thank you